backend/src/api/UserController.ts

```diff
@@ -77,6 +78,11 @@ export default class UserController {
             keyGenerator: (req) => String(req.session.data?.userId)
         });
 
+        const suggestUsernameLimiter = rateLimit({
```

Comment:
I'd suggest to make the limiter per user and use a larger window. Current settings don't prevent from DDOs by constantly sending 10 rps from a single account.